libs/auth/src/inserted_smart_card_auth.ts

```diff
@@ -219,6 +227,21 @@ export class InsertedSmartCardAuth implements InsertedSmartCardAuthApi {
     });
   }
 
+  async logOut(machineState: InsertedSmartCardAuthMachineState): Promise<void> {
+    await this.updateAuthStatus(machineState, { type: 'log_out' });
+  }
```

Comment:
Why we need a log-out method for inserted smart card auth, where logout is typically accomplished by removing your card, is described in a comment later in this commit